### DISCUSSION TITLE
100% on Google Speedpage ?

### DISCUSSION THREAD

**Discussion by coolemur** [ptkdev-boilerplate/svelte-kit-ssr-boilerplate#6] [General]
Hello

I've installed this repo template and ran it on localhost
Google lighthouse is far from 100%

Is there an impact of running this app on localhost? Or am I missing something?

If so, perhaps there should be some kind of note in readme about it, like in what environment we could expect 100%.

Thank you

**Comment by ptkdev** [comment]
@coolemur svelte-kit is an beta/alpha. This version of boilerplate is a WIP.

I am trying to follow the "beta versions" of svelte-kit and update this boilerplate in parallel but it is difficult. 
Wait for the stable version of svelte-kit and this boilerplate will be stable too. 

Unfortunately, I lose scores and features that worked before with every svelte-kit beta update.

**Reply by coolemur** [reply]
Understandable. Thank you for quick response and clarification.

**Comment by ptkdev** [comment]
Updated README.md
